src/components/Login/index.jsx

```diff
@@ -15,7 +19,7 @@ export default function LoginForm() {
     ) {
       history.push('/home');
     } else {
-      setError('Details do not match ....');
+      setError('Details do not match ....');
```

Comment:
I'm weary of this kind of change... 
- You replaced a space with another space
- You're on Windows
- Git & GitHub both think that's important